tests/test_widgets.py

```diff
@@ -464,6 +465,30 @@ def test_no_border(self):
             "                                        \n" +
             "  ------------------------------------  \n")
 
+    def test_no_border_can_scroll(self):
```

Comment:
New tests too!  Brilliant!!  :-)

Just one question...  Any concerns about actually handling user interactions here?  Worth checking a mouse click on the right spot scrolls down?

Reply:
I just assumed as nothing here was touching that code that the existing tests would catch it. Can add more if you think it needed.

Reply:
Probably fine.  It's been a while since I looked at the scroll bars...  Input handling is encompassed in there, so there should be no new path for moving the frame's scrollbar.